### ISSUE TITLE
1.3.0-rc3 ui crash when changing project

### ISSUE THREAD

**Issue by jon-walton** [argoproj/argo-cd#2632]
Checklist:

* [x] I've searched in the docs and FAQ for my answer: http://bit.ly/argocd-faq.
* [x] I've included steps to reproduce the bug.
* [x] I've pasted the output of `argocd version`.

**Describe the bug**

When selecting a project, the UI crashes

**To Reproduce**

In Firefox 70. On the applications page, select a project. The UI will crash with a stacktrace

```text
Something went wrong!

Consider submitting an issue here.

Stacktrace:

onChange@https://argocd.jarvis.tools/main.3d9d0d2fd7b11fc70a02.js:12:230575
onChange@https://argocd.jarvis.tools/main.3d9d0d2fd7b11fc70a02.js:12:212958
Ot</t.prototype.onTagsChange@https://argocd.jarvis.tools/main.3d9d0d2fd7b11fc70a02.js:12:98033
onSelect@https://argocd.jarvis.tools/main.3d9d0d2fd7b11fc70a02.js:12:97237
value/<@https://argocd.jarvis.tools/main.3d9d0d2fd7b11fc70a02.js:80:14646
ia@https://argocd.jarvis.tools/main.3d9d0d2fd7b11fc70a02.js:96:73489
oa@https://argocd.jarvis.tools/main.3d9d0d2fd7b11fc70a02.js:96:73239
Ua@https://argocd.jarvis.tools/main.3d9d0d2fd7b11fc70a02.js:96:83253
Va@https://argocd.jarvis.tools/main.3d9d0d2fd7b11fc70a02.js:96:85729
Os/<@https://argocd.jarvis.tools/main.3d9d0d2fd7b11fc70a02.js:96:98958
t.unstable_runWithPriority@https://argocd.jarvis.tools/main.3d9d0d2fd7b11fc70a02.js:105:3447
Os@https://argocd.jarvis.tools/main.3d9d0d2fd7b11fc70a02.js:96:98891
As@https://argocd.jarvis.tools/main.3d9d0d2fd7b11fc70a02.js:96:98667
Ls@https://argocd.jarvis.tools/main.3d9d0d2fd7b11fc70a02.js:96:98014
Ys@https://argocd.jarvis.tools/main.3d9d0d2fd7b11fc70a02.js:96:99387
Ln@https://argocd.jarvis.tools/main.3d9d0d2fd7b11fc70a02.js:96:30950
```

Refreshing the page will show the applications page with the filter correctly selected.

Clicking the *X* on a selected project will remove the project but log the following in the console:

```text
TypeError: t.labelsFilter is undefinedapplications-list.tsx:213:40
    onChange applications-list.tsx:213
    onChange applications-filter.tsx:139
    onTagsChange tags-input.tsx:102
    onClick tags-input.tsx:33
    React 12
    unstable_runWithPriority scheduler.production.min.js:18
    React 2
    Ln self-hosted:1063
```

**Expected behavior**

Don't crash

**Version**

```shell
argocd: v1.3.0-rc3+e0de330
  BuildDate: 2019-10-29T19:43:40Z
  GitCommit: e0de3300d23d559ae1248749e1c351bda684c013
  GitTreeState: clean
  GoVersion: go1.12.6
  Compiler: gc
  Platform: linux/amd64
```

**Comment by alexmt** [comment]
Thank you for reporting it! I think bug supposed to be fixed by https://github.com/argoproj/argo-cd/pull/2619

I've just published v1.3.0-rc4 which includes the fix: https://github.com/argoproj/argo-cd/releases/tag/v1.3.0-rc4 . Can you please upgrade and confirm the issue disappears for you?

**Comment by jon-walton** [comment]
yep fixed in rc4. thanks!
